OrbitGrpcProtos/capture.proto

```diff
@@ -226,9 +226,10 @@ message AddressInfo {
 }
 
 message ModulesUpdateEvent {
+  reserved 3;
   int32 pid = 1;
   uint64 timestamp_ns = 2;
-  repeated ModuleInfo modules = 3;
+  ModuleInfo module = 4;
```

Comment:
Given that we don't need compatibility with different versions here, can we keep the field numbers clean instead of already having `reserved` numbers?

Reply:
well, numbers are free - this is good practice and we should probably do this by default even if this is not really needed at this point. There are 0 downsides to this.

Reply:
Well, there is one downside to this: it litters the proto. I don't like leaving litter behind when it's not necessary, even if it's a good (or necessary) practice in a different context.

Also this is now basically a new proto as it has different semantics (it was carrying the entire parsed maps before, it carries a single new module now, and indeed it made sense to rename it because of that), so it makes even less sense to have a field number already obsolete.